src/events/api/views.py

```diff
@@ -62,9 +62,110 @@ def get(self, request):
         return Response(response_data)
 
 
-class ScheduleAPIView(RetrieveAPIView):
-    authentication_classes = [TokenAuthentication]
-    permission_classes = [IsAuthenticated]
+
+class EventWrapper:
+
+    def __init__(self, obj):
+        self.obj = obj
+
+    @property
+    def event_id(self) -> int:
+        return self.obj.id
+
+    @property
+    def event_type(self) -> str:
+        TYPE_MAP = {
+            CustomEvent: 'custom',
+            KeynoteEvent: 'keynote',
+            ProposedTalkEvent: 'talk',
+            SponsoredEvent: 'sponsored',
+            ProposedTutorialEvent: 'tutorial',
+        }
+        return TYPE_MAP[type(self.obj)]
+
+    @property
+    def title(self) -> str:
+        if isinstance(self.obj, KeynoteEvent):
+            return self.obj.session_title
+        elif isinstance(self.obj, (ProposedTalkEvent, ProposedTutorialEvent)):
+            return self.obj.proposal.title
+        else:
+            return self.obj.title
+
+    @property
+    def speakers(self) -> str:
```

Comment:
return as a list of speaker names

Reply:
Fixed, now speakers will return a list of str